Frontend/Pages/Projects/Index.cshtml

```diff
@@ -1,8 +1,9 @@
 ﻿@page "/project/{urn}"
+@using Frontend.ExtensionMethods
 @model Frontend.Pages.Projects.Index
 
 @{
-    ViewBag.Title = Model.OutgoingAcademyName;
+    ViewBag.Title = Model.IncomingTrustName;
```

Comment:
V minor but might be worth checking with Rinto if this needs to be title case to match the h1 on the page

Reply:
Good spot, it does need to be, Rinto and Ady mentioned it. I think I have this change ready to push i'll check